content/blog/entries/gsoc2019-arimadian/contents.lr

```diff
@@ -0,0 +1,28 @@
+title: Ari Madian - My GSoC 2019 Blog
+---
+categories:
+GSoC
+Google Summer of Code
+cc-chooser
+cc-license-chooser
+---
+author: akmadian
+---
+pub_date: 2019-06-11
+---
+body:
+
+**Week 2:**
```

Comment:
Could you add a link to your project description or just describe the project briefly so that readers can have some context?